gridstatus/utils.py

```diff
@@ -219,14 +219,19 @@ def download_csvs_from_zip_url(url, process_csv=None, verbose=False):
     z = get_zip_folder(url, verbose=verbose)
 
     all_dfs = []
+
     for f in z.filelist:
         if f.filename.endswith(".csv"):
             df = pd.read_csv(z.open(f.filename))
             if process_csv:
                 df = process_csv(df, f.filename)
+
+            # Some data files have leading whitespace in header - remove it
+            df = df.rename(columns=lambda x: x.strip())
```

Comment:
Does this have effects on other places that use this util?

Reply:
That's a good question and I don't know. Maybe I should add a param to control this so it defaults to the current behavior instead? Without this here the concat gets messed up and we don't get a clean dataframe, so we somehow need to handle this new wrinkle.

Reply:
Yeah, adding another param to not change the existing behavior seems like the right approach.